mercure.rst

```diff
@@ -693,6 +659,51 @@ Enable the panel in your configuration, as follows:
 
 .. image:: /_images/mercure/panel.png
 
+Async dispatching
+-----------------
+
+.. tip::
+
+    Async dispatching is discouraged. Most Mercure hubs already
+    handle publications asynchronously and using Messenger is
+    usually not necessary.
+
+Instead of calling the ``Publisher`` service directly, you can also let Symfony
+dispatching the updates asynchronously thanks to the provided integration with
+the Messenger component.
+
+First, be sure :doc:`to install the Messenger component </messenger>`
+and to configure properly a transport (if you don't, the handler will
+be called synchronously).
+
+Then, dispatch the Mercure ``Update`` to the Messenger's Message Bus,
+it will be handled automatically::
+
+    // src/Controller/PublishController.php
+    namespace App\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\Mercure\Update;
+    use Symfony\Component\Messenger\MessageBusInterface;
+
+    class PublishController extends AbstractController
+    {
+        public function publish(MessageBusInterface $bus): Response
+        {
+            $update = new Update(
+                'http://example.com/books/1',
+                json_encode(['status' => 'OutOfStock'])
+            );
+
+            // Sync, or async (Doctrine, RabbitMQ, Kafka...)
+            $bus->dispatch($update);
+
+            return new Response('published!');
+        }
+    }
```

Comment:
maybe we can remove this entirely?

IIRC, the message handler is deprecated, and is not registered by default.

Reply:
Actually, we didn't deprecate the handler (https://github.com/symfony/mercure/blob/main/src/Messenger/UpdateHandler.php), so we should keep this section.